docs/issue_template.md

```diff
@@ -0,0 +1,35 @@
+# Issue Template
+
+## Context
+
+Please provide any relevant information about your setup. This is important in case the issue is not reproducible except for under certain conditions.
+
+* Firmware Version:
```

Comment:
Some of these aren't relevant  to the technologies in this project. Here's a suggestion (kept SDK version in the case that we add NativeScript):

```
* Operating System:
* Browser Version:
* Node Version:
* SDK Version:
```

Reply:
I removed all but SDK and Browser. We'll definitely want to know bugs in major browsers.